src/components/common/CategoriesCards/CategoryCard.jsx

```diff
@@ -0,0 +1,38 @@
+import { memo } from 'react';
+import { cn } from '../../../utilities/cn';
+
+const CategoryCard = ({
+  title,
+  className,
+  isActive,
+  backgroundImage,
+  icon,
+  onClick,
+}) => {
+  return (
+    <button
+      className={cn(
+        'flex h-48 w-[8.625rem] transform rounded-lg transition duration-300 ease-in-out hover:text-orange-color hover:shadow-xl',
+        isActive
+          ? 'scale-105 rounded-lg text-orange-color shadow-[0_0_15px_rgba(255,125,0,0.6)] ring-1 ring-inset ring-orange-color'
+          : 'text-dark-color',
+
+        className,
+      )}
+      onClick={onClick}
+      aria-label={`A button that sorts city guides by parameter ${title}`}
```

Comment:
just for the extra word about a11y

I recently found out that there is an `aria-pressed` attribute, especially for this case. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-pressed

Your button behaves like a toggle button but there are no indicators of that for the users who use your app using assistive technologies like a VoiceOver etc. 

So you can apply this attribute to improve the a11y of your site a little.

Reply:
Don't forget that the "Without category" should not have this attribute because it behaves as usual button